pkgs/applications/science/math/mathematica/default.nix

```diff
@@ -35,9 +35,25 @@
 , xorg
 , zlib
 , lang ? "en"
+, cudaSupport ? false
+, nvidia_x11 ? null
+, cudatoolkit ? null
```

Comment:
```suggestion
, nvidia_x11
, cudatoolkit
```

Reply:
```diff
- , cudaSupport ? false
+ , config
+ , cudaSupport ? config.cudaSupport or false
```

...here or (@SuperSandro2000 ? Grep shows both approaches) in top-level for consistency with `import nixpkgs { config.cudaSupport = true; }`.

Reply:
Yeah sure but not sure if the config is required. I am guessing not.

Reply:
https://github.com/NixOS/nixpkgs/blob/e2f381b2f1879ad6d68be5a9e9e6e3288529bfaf/pkgs/applications/misc/blender/default.nix#L8
https://github.com/NixOS/nixpkgs/search?q=config.cudasupport